search/processor.cpp

```diff
@@ -644,22 +633,15 @@ void Processor::InitParams(QueryParams & params)
       if (type >= 0)
       {
         ASSERT(type == 70 || type > 4000, (type));
```

Comment:
Could you please explain why type == 70 and type > 4000 are correct for this case?